### ISSUE TITLE
PSNR of moving objects in Table 1

### ISSUE THREAD

**Issue by YikangZhang1641** [zju3dv/street_gaussians#89]
Thank you for sharing the code.

I trained all scripts in experiments_waymo. PSNR/SSIM/LPIPS generated from metrics.py are the same as Table 1.
But there's no "PSNR*" in metrics.py, so I simply added a few lines to calculate psnr() with object masks:

        obj_bounds = []
        for cam_info in tqdm(cam_infos, desc="Reading image progress"):
            obj_bound_path = os.path.join(dataset.source_path, "dynamic_mask", f'{image_name}.png')
            obj_bound = Image.open(obj_bound_path).resize(gt.size, Image.NEAREST)
            obj_bounds.append(tf.to_tensor(obj_bound)[:1, :, :].bool())

        psnrs_obj = []
        for idx in tqdm(range(len(renders)), desc="Metric evaluation progress"):
            obj_mask = obj_bounds[idx].cuda()
            if obj_mask.sum() != 0:
                psnrs_obj.append(psnr(render, gt, obj_mask))
        print("  PSNR(OBJ) : {:>12.7f}".format(torch.tensor(psnrs_obj).mean(), ".5"))

However, the result of PSNR* is different from Table 1:
![Image](https://github.com/user-attachments/assets/baa4722c-9800-45e4-8c56-9851b05c7a78)

so I wonder what's the implementation of PSNR*. Could you update it in metrics.py?

**Comment by yunzhiy** [comment]
The length and width dimensions of object tracklets are scaled by a factor of 1.5 to fully cover the moving actor, as mentioned in the supplementary material. You may need to adjust `data.box_scale` for each sequence to match the experiment setting.

**Comment by shoubuliaol** [comment]
hi, may i ask if you calculate these metrics through the command "python render.py --config configs/xxxx.yaml mode evaluate", i want to do this too, but my test camera always fails to load"Number of test cameras: 0
Rendering Testing View: 0it [00:00, ?it/s]
"

<img width="1175" height="498" alt="Image" src="https://github.com/user-attachments/assets/cb9747ea-7c2e-4e69-b4c4-09b938a33009" />and under output file where test is empty.
